index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    
+    <section id = "welcome-section"> 
+      <h1> Hi in this section your are going to view my portfolio </h1>
+       </section>>
+       <section id = "projects">
+         <article class = "project-tile">Projects</article>
+         <a href="file:///C:/Users/MILI/Documents/Microverse/Colab-2/index.html">First Project</a>
+       </section>
+       <nav id="navbar">
+         <ul>
+           <li><a href="#projects">First Project</a></li>
+         </ul>
+       </nav>
+       <a id="profile-link" target="_blank" href="https://github.com/emiliazm/Cola-3">Git Hub link</a>
```

Comment:
For readability, kindly add blank lines to separate logical code blocks.

Reply:
- [ ] Nicely done 👍